crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -760,6 +680,106 @@ pub fn view_3d(
     Ok(())
 }
 
+/// Show center of orbit camera when interacting with camera (it's quite helpful).
+fn show_view_eye_center(
```

Comment:
```suggestion
fn show_orbit_eye_center(
```

this one is specifically about the orbit camera